### ISSUE TITLE
VirtaulMachine.getConfig() is throwing RuntimeException

### ISSUE THREAD

**Issue by atokasema** [yavijava/yavijava#57]
VirtaulMachine.getConfig() is throwing Runtime Exception.
Exception detail below:
java.lang.RuntimeException: java.rmi.RemoteException: Exception in SoapClient.invoke:; nested exception is: 
    java.lang.NoSuchFieldException: faultToleranceType

**Comment by michaelrice** [comment]
Hello @atokasema can you provide your version of vSphere you are hitting, and also what version of yavijava you are using, and what version of java are you using?

**Comment by atokasema** [comment]
vSphere 6.0
How do i check the yavijava version?
Java 7

**Comment by michaelrice** [comment]
@atokasema the jar file has the version in the name in most cases. Where did you get the jar you are running now?

**Comment by atokasema** [comment]
Just to verify my setup, I used vijava55.jar and tried the same call on same setup and it worked fine.

jar file was downloaded from this website only, but by my colleague and passed on to me. yavijava60.jar is jar file I am using.

jar MANIFEST file content:
Manifest-Version: 1.0
Ant-Version: Apache Ant 1.9.4
Bnd-LastModified: 1428963351169
Bundle-ManifestVersion: 2
Bundle-Name: yavijava60
Bundle-SymbolicName: yavijava60
Bundle-Version: 0
Created-By: 1.7.0_75 (Oracle Corporation)

**Comment by michaelrice** [comment]
@atokasema This is not a bug. Please see #29 support for 6.0 is in progress. The work has not been done yet for this to work.

**Comment by michaelrice** [comment]
Specifically this will need to be implemented: http://pubs.vmware.com/vsphere-60/index.jsp?topic=%2Fcom.vmware.wssdk.apiref.doc%2Fvim.VirtualMachine.FaultToleranceType.html

You should really be using yavijava 5.5.10 from public maven. The vsphere 6.0 branch should only be used right now by developers working to help add vsphere 6.0 support or to report bugs about the features identified in #29 that have been implemented.

**Comment by atokasema** [comment]
I got some time to debug it and I was able to root cause the issue.
vSphere 6 SDK added a new filed called "vmxConfigChecksum" of type base64Binary in VirtualMachineConfigInfo data object. The code generator is converting this base64Binary to byte array  but when we get this value in soap response, it is coming as a single value like string and not an array of values. The marshaling is trying to convert the whole base64Binary value to one byte and its throwing error. 
We need to add fix to handle this base64Binary data type
